HeliPort/NetworkManager.swift

```diff
@@ -287,3 +289,34 @@ final class NetworkManager {
         return ITL80211_SECURITY_UNKNOWN
     }
 }
+
+extension NetworkManager {
+    // from Goshin
+    private static let ipAddressRegex = #"\s([a-fA-F0-9\.:]+)(\s|%)"# // for ipv4 and ipv6
```

Comment:
Wouldn't it be better to declare it as a variable in `getRouterAddress`? as it is not a generic pattern for IP address, only useful for parsing `netstat -rn`.

Reply:
You're right. Let me remove that line and place it in `getRouterAddress`.